invenio_cli/commands/local.py

```diff
@@ -46,22 +47,24 @@ def _copy_statics_and_assets(self):
         """Copy project's statics and assets into instance dir."""
         click.secho("Copying project statics and assets...", fg="green")
 
-        static = "static"
-        src_dir = self.cli_config.get_project_dir() / static
-        src_dir = str(src_dir)  # copy_tree below doesn't accept Path objects
-        dst_dir = self.cli_config.get_instance_path() / static
-        dst_dir = str(dst_dir)
-        # using it for a different purpose then intended but very useful
-        copy_tree(src_dir, dst_dir)
-
-        assets = "assets"
-        src_dir = self.cli_config.get_project_dir() / assets
-        src_dir = str(src_dir)
-        dst_dir = self.cli_config.get_instance_path() / assets
-        dst_dir = str(dst_dir)
-        # The full path to the files that were copied is returned
-        copied_files = copy_tree(src_dir, dst_dir)
-        return copied_files
+        if os.path.isdir("static"):
```

Comment:
Why this check `isdir` is needed?

Reply:
We will have to run the `invenio-cli install `command, which will perform this copying of non-existing in ILS static and assets folders